src/components/ImageVote.vue

```diff
@@ -132,26 +154,7 @@ export default {
           this.imageLeftUrl = responseData[0].Url
           this.imageRight = responseData[1].id
           this.imageRightUrl = responseData[1].Url
-        }).catch(error => {
-          console.warn(error.message)
-          let self = this
-          this.$vs.dialog({
-            type: 'confirm',
-            color: 'danger',
-            title: `Verbindungsfehler`,
-            text: 'Zurzeit kann keine Verbindung hergestellt werden. Überprüfen Sie bitte das Netzwerk und versuchen Sie es später erneut.',
-            acceptText: 'Bestätigen',
-            cancelText: 'Abbrechen',
-            accept: function () {
-              self.voteCount--
-              self.voteTotal--
-              self.nextImagePair()
-            },
-            cancel: function () {
-              self.$router.push({ name: 'start' })
-            }
-          })
-        })
+        }).catch((error) => this.promptBadImage(error))
```

Comment:
How about simply using a `this.pauseVoting` variable to track the state of this prompt?